reference/conanfile/attributes.rst

```diff
@@ -1670,3 +1670,66 @@ When ``True`` it enables the new run in a subsystem bash in Windows mechanism. :
     class FooRecipe(ConanFile):
         ...
         win_bash = True
+
+.. _test_type:
+
+test_type
+---------
+
+.. warning::
+
+    Test type is an **experimental** feature subject to breaking changes in future releases.
+
+Available since: `1.44.0 <https://github.com/conan-io/conan/releases/tag/1.44.0>`_
+
+This attribute allows testing requirements and build requiments explicitly on test package.
+In Conan 2.0 the ``test_type`` attribute will be ignored, the behavior will be always explicit, so declaring ``test_type="explicit"`` will make the test recipe compatible with Conan 2.0.
+The possible values are:
+
+ - ``requires`` (by default): It will consume the tested requirement automatically as usual.
+ - ``build_requires``: It will consume the test build requirement automatically. It can be combined with ``requires``.
+ - ``explicit``: The test package will not solve its dependencies, need to declare explicitly with ``self.tested_reference_str``. This will be the default behavior for Conan 2.0. The ``explicit`` can not be combined with ``requires`` neither ``build_requires``, it doesn't generate an error, but only ``explicit`` will be computed.
```

Comment:
```suggestion
 - ``explicit``: The test package will not solve its dependencies automatically, you need to declare it explicitly using the reference at ``self.tested_reference_str``. This will be the only behavior for Conan 2.0. The additional values ``requires`` and ``build_requires`` (if specified) will be ignored.
```